code/notificationbuilder/src/main/java/com/adobe/marketing/mobile/notificationbuilder/internal/builders/MultiIconNotificationBuilder.kt

```diff
@@ -129,6 +129,7 @@ internal object MultiIconNotificationBuilder {
                     R.id.icon_item_image_view,
                     interactionUri,
                     null,
+                    null,
```

Comment:
nit: In L73 in this file, a dismiss action is created for close button click. Can we refactor it to use `setRemoteViewClickAction` and pass in the dismiss action?

Reply:
good callout, i've refactored the code at L73 to use `RemoteViews.setRemoteViewClickAction` as well